packages/dialect-react-ui/components/Chat/screens/Main/pages/ThreadPage/Thread.tsx

```diff
@@ -35,9 +36,18 @@ export default function Thread() {
         .catch(handleError);
     }
   };
-  const youCanWrite = dialect?.dialect.members.some(
-    (m) => m.publicKey.equals(wallet?.publicKey) && m.scopes[1]
-  );
+
+  useEffect(() => {
+    const membersContainCurrentKey = dialect?.dialect.members.some(
+      (m) => m.publicKey.equals(wallet?.publicKey) && m.scopes[1]
```

Comment:
Could you please add a comment about m.scopes[1], a simple explanation why [1] would be sufficient. This will help us understand later on as we develop. Thanks!